### DISCUSSION TITLE
LGTM: Cannot Spot Issue in Checkout

### DISCUSSION THREAD

**Discussion by ax3l** [community/community#24336] [Apps, API and Webhooks]
<p>Hi,</p>
<p>since a few days, my C++ integrations with <a href="http://LGTM.com" rel="noopener nofollow ugc">LGTM.com</a> fail with a “checkout” issue on several of my repos.<br>
Example: <a href="https://github.com/openPMD/openPMD-api" rel="noopener nofollow ugc">https://github.com/openPMD/openPMD-api</a></p>
<p>I cannot spot a recent blog entry or change, e.g., in the Ubuntu image that LGTM uses. Looking at the log, the return code for checkout seems to be zero (no error), leaving me puzzled.<br>
<a href="https://lgtm.com/projects/g/openPMD/openPMD-api/logs/rev/pr-1c78fd7a97b81ba78e627a105a24bc8832f648d6/lang:cpp/stage:Build%20merge_d824f9f3abda0f7a98f0d0dedc1b540297575aae" rel="noopener nofollow ugc">https://lgtm.com/projects/g/openPMD/openPMD-api/logs/rev/pr-1c78fd7a97b81ba78e627a105a24bc8832f648d6/lang:cpp/stage:Build%20merge_d824f9f3abda0f7a98f0d0dedc1b540297575aae</a></p>
<p>Does anyone know how  to further debug the <code>Failed with bad exit code during 'Checkout'</code> error? The error really does not look obvious to me.</p>
<p>Thanks for your help,<br>
Axel</p>

**Comment by Irving-cl** [comment]
<p>I experienced the same problem and got stuck there.</p>

**Comment by utilForever** [comment]
<p>I experienced the same problem, too. :&lt;</p>

**Comment by ice0** [comment]
<p>Same here: <a href="https://lgtm.com/projects/g/synfig/synfig/logs/rev/f25b03e06407558c9b49dc53ac0709bc095657ba/lang:cpp/stage:Build%20child_f25b03e06407558c9b49dc53ac0709bc095657ba" rel="noopener nofollow ugc">https://lgtm.com/projects/g/synfig/synfig/logs/rev/f25b03e06407558c9b49dc53ac0709bc095657ba/lang:cpp/stage:Build%20child_f25b03e06407558c9b49dc53ac0709bc095657ba</a></p>

**Comment by ice0** [comment]
<p>All works fine now, thank you!</p>

**Comment by ax3l** [comment, marked as answer]
<p>Output is now more verbose, my issue was rooted in an too old version of CMake on the Ubuntu Eoan image that is running the C++ tests.</p>
